kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -190,6 +190,12 @@
       userIsLearner() {
         return this.getUserKind == UserKinds.LEARNER;
       },
+      showSoudNotice() {
+        return (
+          this.isSubsetOfUsersDevice &&
+          [UserKinds.ADMIN, UserKinds.COACH].includes(this.getUserKind)
```

Comment:
Can maybe just use `isAdmin || isCoach` from the getters here: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/assets/src/state/modules/session.js#L18

That means we are showing the warning to superusers too, which seems relevant.

Reply:
:+1: updated!